app/src/main/admin/components/ModellingGroups/List/ModellingGroupsList.tsx

```diff
@@ -10,7 +10,7 @@ interface ModellingGroupsProps extends RemoteContent {
     groups: ModellingGroup[]
 }
 
-export class ModellingGroupsListComponent extends RemoteContentComponent<ModellingGroupsProps> {
+export class ModellingGroupsListComponent extends RemoteContentComponent<ModellingGroupsProps, {}> {
```

Comment:
In general, we have been using `undefined` rather than `{}` in these kind of cases.

Reply:
* wrote 2 unit tests: 1 tests on download component and 1 on onetimebutton
* changed {} to undefined